packages/common/test/services/coingecko/CoingeckoClient.test.ts

```diff
@@ -0,0 +1,165 @@
+import { expect } from 'earljs'
+import { Response } from 'node-fetch'
+
+import {
+  CoingeckoClient,
+  CoingeckoId,
+  CoingeckoMarketChartRangeParams,
+  HttpClient,
+  mock,
+} from '../../../src'
+import { MOCK_DATA } from './MOCK_DATA'
+
+describe.only('CoingeckoClient', () => {
```

Comment:
Don't include `.only`. Also, move to the new describe style: `describe(CoingeckoClient.name)`